_viash.yaml

```diff
@@ -81,12 +81,7 @@ config_mods: |
   .runners[.type == "nextflow"].config.labels := { lowmem : "memory = 20.Gb", midmem : "memory = 50.Gb", highmem : "memory = 100.Gb", lowcpu : "cpus = 5", midcpu : "cpus = 15", highcpu : "cpus = 30", lowtime : "time = 1.h", midtime : "time = 4.h", hightime : "time = 8.h", veryhightime : "time = 24.h" }
 
 repositories:
-  - name: openproblems-v2
+  - name: openproblems
     type: github
-    repo: openproblems-bio/openproblems-v2
-    tag: main_build
-  - name: core
-    type: github
-    repo: openproblems-bio/core
-    tag: build/main
-    path: viash/core
+    repo: openproblems-bio/openproblems
```

Comment:
Should this to the task repository or the main one?

Reply:
Should be the main one since we use some dependencies from the main repository in the workflows.